content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -145,9 +148,10 @@ For detailed PowerFlex installation procedure, see the [Dell PowerFlex Deploymen
 
    | Parameter | Description | Required | Default |
    | --------- | ----------- | -------- |-------- |
+   | namespace | Specifies namespace where the driver will be installed | Yes | "vxflexos" |
    | dnsPolicy | Determines the DNS Policy of the Node service | Yes | ClusterFirstWithHostNet |
-   | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType`. In OCP <= 4.16 and K8s <= 1.29, fsGroupPolicy is an immutable field. | No | "ReadWriteOnceWithFSType" |
-   | replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, excess pods will become stay in a pending state. Defaults are 2 which allows for Controller high availability. | Yes | 2 |
+   | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType`. In OCP <= 4.16 and K8s <= 1.29, fsGroupPolicy is an immutable field. | No | "File" |
```

Comment:
these changes applicable to previous version of docs ? if yes can you use the same PR for updating previous v1,v2 etc ?

Reply:
Yes @prablr79 , great catch, update the parameters for v1, v2, v3 versions also.

Reply:
Please review

Reply:
Some fields were missing in the previous versions, updated them also.